index.js

```diff
@@ -67,10 +67,12 @@ fs.readFile(userHomeDir + '/.boxrc.json', 'utf8', async function(err, data) {
       if(l) {
         await getDetailedMessages(currentChannel, session)
       }
-      if(c) {
+      else if(c) {
```

Comment:
Let's align the `else if..` and `else` with its proceeding `if` block ex:

```javascript
if (l) {
  // some code
} else if (c) {
  // some code
} else {
  // some code
}
```

I'll add a task to configure ESLint so we can catch these prior to pushing code!

Reply:
It would also be great to add spaces around the condition inside parentheses. For example, `if (l)` instead of `if(l)`

Reply:
The syntax has been corrected.